src/hooks/screen/useChatScreenHook.ts

```diff
@@ -73,12 +99,18 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           userId,
           selectedChannel?.id,
           message,
-          localDb
+          localDb,
+          replyData ? MESSAGE_TYPE_REPLY : MESSAGE_TYPE_REGULAR,
+          'pending',
+          json
         );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function has been updated to handle reply messages and anonymous messages. Ensure that the logic correctly handles these cases.
```diff
+           localDb,
+           replyData ? MESSAGE_TYPE_REPLY : MESSAGE_TYPE_REGULAR,
+           'pending',
+           json
+          );
+          currentChatSchema.save(localDb);
+          refresh('chat');
+          refresh('channelList');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->